server/models/Item.js

```diff
@@ -1,14 +1,9 @@
 const mongoose = require('mongoose')
 const ItemSchema = new mongoose.Schema({
-    itemID: {
-        type: String,
-        default: Schema.Types.ObjectId,
-    },
     itemName: {
```

Comment:
You're already on the Item collection. You don't really need to write "itemName". We know it's the name of the item.

don't you think so?

Reply:
I'll fix it.